legion/engine/rendering/util/bindings.hpp

```diff
@@ -28,15 +29,22 @@
 
 /* uniform 15 */  #define SV_SCENECOLOR     SV_LIGHTCOUNT + 1
 /* uniform 16 */  #define SV_SCENEDEPTH     SV_SCENECOLOR + 1
-/* uniform 17 */  #define SV_HDROVERDRAW    SV_SCENEDEPTH + 1
+/* uniform 17 */  #define SV_SCENENORMAL    SV_SCENEDEPTH + 1
+/* uniform 18 */  #define SV_SCENEPOSITION  SV_SCENENORMAL + 1
+/* uniform 19 */  #define SV_HDROVERDRAW    SV_SCENEPOSITION + 1
 
-/* uniform 21 */  #define SV_ALBEDO         SV_HDROVERDRAW + 4   
-/* uniform 22 */  #define SV_NORMALHEIGHT   SV_ALBEDO + 1   
-/* uniform 23 */  #define SV_MRDAO          SV_NORMALHEIGHT + 1    
-/* uniform 24 */  #define SV_EMISSIVE       SV_MRDAO + 1        
-/* uniform 25 */  #define SV_HEIGHTSCALE    SV_EMISSIVE + 1
+/* uniform 23 */  #define SV_ALBEDO         SV_HDROVERDRAW + 4   
+/* uniform 24 */  #define SV_NORMALHEIGHT   SV_ALBEDO + 1   
+/* uniform 25 */  #define SV_MRDAO          SV_NORMALHEIGHT + 1    
+/* uniform 26 */  #define SV_EMISSIVE       SV_MRDAO + 1        
+/* uniform 27 */  #define SV_HEIGHTSCALE    SV_EMISSIVE + 1
 
-/* uniform 21 */  #define SV_MATERIAL       SV_ALBEDO
+/* uniform 23 */  #define SV_MATERIAL       SV_ALBEDO
+
+/* attachment 0 */ #define FRAGMENT_ATTACHMENT  GL_COLOR_ATTACHMENT0
```

Comment:
I do not perpendicularly like this name as a substitute for GL_COLOR_ATTACHMENT0, but it might make sense than I realize

Reply:
we had a call about this